torch/csrc/jit/codegen/cuda/fusion.cpp

```diff
@@ -31,10 +31,9 @@ void ExprSort::handle(Expr* expr) {
 std::vector<Expr*> ExprSort::getExprs(
     Fusion* fusion,
     bool from_outputs_only,
-    bool breadth_first,
-    bool respect_compute_at) {
+    bool breadth_first) {
```

Comment:
I thought we got rid of breadth_first? In any case, would you mind removing it part of this PR?

Reply:
That one has been on my to-do list. Please see #207.